### DISCUSSION TITLE
Need support to develop MODBUS RTU device service

### DISCUSSION THREAD

**Discussion by georgethomas08** [edgexfoundry/community#175] [Q&A]
Hi Team,

I need support to develop a device service to poll the data from a MODBUS slave device. It would be grateful if you could share complete document of device service step by step. Website documentation is not helping for beginners

**Comment by cloudxxx8** [comment]
The document is step by step
https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSDK-Go/

**Comment by jiekechoo** [comment]
As @cloudxxx8 mentioned, the SDK document is good for you if you need to develop a new MODBUS device service.
If you are using a normal/standard MODBUS slave device, try `device-modbus` micro service, maybe it could help. https://docs.edgexfoundry.org/3.0/examples/Ch-ExamplesAddingModbusDevice/
